packages/lodestar-beacon-state-transition/src/fast/signatureSets/types.ts

```diff
@@ -0,0 +1,18 @@
+import {PublicKey} from "@chainsafe/bls";
+import {BLSSignature, Root} from "@chainsafe/lodestar-types";
+
+export type ISignatureSet = ISignatureSinglePubkeySet | ISignatureMultiplePubkeySet;
+
+interface ISignatureSinglePubkeySet {
+  type: "single-pubkey";
+  pubkey: PublicKey;
+  signingRoot: Root;
+  signature: BLSSignature;
+}
+
+interface ISignatureMultiplePubkeySet {
+  type: "multiple-pubkeys";
```

Comment:
maybe make this enum. Also I think single/aggregate/multiple should be better names

Reply:
Damn right, much more cleaner